packages/patrol_cli/lib/src/android/android_test_backend.dart

```diff
@@ -186,9 +188,44 @@ class AndroidTestBackend {
   Future<void> execute(
     AndroidAppOptions options,
     Device device, {
+    String? flavor,
     bool interruptible = false,
+    required bool showFlutterLogs,
+    required bool showTestSteps,
   }) async {
     await _disposeScope.run((scope) async {
+      // Read patrol logs from logcat
+      final processLogcat = await _processManager.start(
+        [
+          'adb',
+          'shell',
+          'logcat',
+          '-T',
+          '1',
+          'PatrolServer:I Patrol:I flutter:I *:S',
+        ],
+        runInShell: true,
+      )
+        ..disposedBy(scope);
+
+      var flavorPath = '';
+      if (flavor != null) {
+        flavorPath = 'flavors/$flavor/';
+      }
+      final reportPath =
+          'file://${_rootDirectory.childDirectory('android').path}/build/app/reports/androidTests/connected/${flavorPath}index.html';
```

Comment:
This one probably won't work on Windows (due to different separated used there).